torch/csrc/jit/codegen/cuda/arith.h

```diff
@@ -553,6 +553,13 @@ TORCH_CUDA_CU_API TensorView* index_select(
     TensorView* input,
     int dim,
     TensorView* index);
+// torch.gather
+TORCH_CUDA_CU_API TensorView* torch_gather(
```

Comment:
We may want to rename the existing nvfuser gather to something else, but that can be done later.